iFixit/react-commerce
# Strapi SDK: Generate graphql schema

Implements a workaround for the zod input schema becoming `definedNonNullAnySchema` due to the graphql endpoint being `http`, not `https`. Described in more detail here: https://github.com/Code-Hex/graphql-codegen-typescript-validation-schema/issues/131#issuecomment-1344439278

This is an alternative solution to https://github.com/iFixit/react-commerce/pull/1709

Closes #1708 

## QA

Run `pnpm dev`. Make sure that no new diff is generated by codegen, and that all the codegen logs indicate that it's successfully running.

CC @federicobadini @danielbeardsley
